pom.xml

```diff
@@ -34,8 +34,9 @@
         <url>https://github.com/${gitHubRepo}</url>
     </scm>
     <properties>
-        <revision>2.4.1</revision>
-        <changelist>-SNAPSHOT</changelist>
+        <revision>3</revision>
+        <changelist>999999-SNAPSHOT</changelist>
+        <hpi.compatibleSinceVersion>3</hpi.compatibleSinceVersion>
```

Comment:
```suggestion
```
This is not needed for CD, nor needed in this case. There are no incompatible config changes users can not upgrade from.

Reply:
there was an incompatibility for ver 2.0.0, and as now the versioning changes i wanted to restore this (I missed this here: https://github.com/jenkinsci/zos-connector-plugin/pull/19/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8

Reply:
Namely, this line: https://github.com/jenkinsci/zos-connector-plugin/blob/02b64c4947aa240bcf42d9e025b6eb12188e3531/pom.xml#L62